### ISSUE TITLE
[EN][Quest][Hellfire Peninsula] Birds Of A Feather

### ISSUE THREAD

**Issue by Enay** [Atlantiss/BugTracker#3132]
In the quest **Birds Of A Feather** (http://www.wowhead.com/quest=9397), you can't interact with **Kaliri Nest** (http://www.wowhead.com/object=181582), which after interaction should spawn **Female Kaliri Hatchling** (http://www.wowhead.com/npc=17034) that you need to capture for the quest.

Quest can still be completed because there is a **Female Kaliri Hatchling** already spawned, without any interaction with the **Kaliri Nest**(s). This is not blizzlike, though.

**Comment by Crusaderz** [comment]
Quest is not completable anymore. Spawned Female Kaliri Hatchling is aggresive and  [Empty Birdcage] cannot be used on her.

**Comment by Samineh** [comment]
Yep, this is  still bugged. I can confirm that. The female Kaliri Hatchlings are hostile and when you try to capture them this error will appear: "you can't do that yet"

**Comment by WilJames** [comment]
Still not working as it should.
